lib/tasks/test_images.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+desc 'Test image downloads for large pixel size images'
```

Comment:
I'm happy to 🔥 this commit if we want. I found this useful to quickly gather some data.